src/components/UI/TokenSelector/TokenSelector.js

```diff
@@ -3,15 +3,19 @@ import React from 'react';
 
 import {ReactComponent as DownArrowIcon} from '../../../assets/svg/icons/collapse.svg';
 import {useColors} from '../../../hooks';
+import {toClasses} from '../../../utils';
 import {CryptoLogoSize} from '../CryptoLogo/CryptoLogo';
 import {CryptoLogo, Icon} from '../index';
 import styles from './TokenSelector.module.scss';
 
-export const TokenSelector = ({tokenData, onClick}) => {
+export const TokenSelector = ({isDisabled, tokenData, onClick}) => {
```

Comment:
`tokenData, isDisabled, onClick`

Reply:
we changed the design of what needs to be done, so resolve